sapphire/tests/esd_load_data.py

```diff
@@ -29,11 +32,35 @@ def perform_load_data(filename):
         esd.load_data(datafile, '/', weather_source, 'weather')
 
 
+def perform_esd_download_data(filename):
+    """Load data from esd/api to h5 (filename)"""
+
+    filters = tables.Filters(complevel=1)
+    start = datetime.datetime(2012, 01, 01, 0, 0, 0)
```

Comment:
@153957: Ouch! Thanks for the heads up. It doesn't seem to matter here, but I'll change it anyway.